src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movies {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <>
+    {movies.map((movie) => (
+      <div className="movies">
+        <MovieCard
+          movie={movie}
+          key={movie.imdbId}
+        />
+      </div>
+    ))}
+  </>
```

Comment:
This creates a list for each movie, but you should only have one list
```suggestion
  <div className="movies">
    {movies.map((movie) => (
      <MovieCard
        movie={movie}
        key={movie.imdbId}
      />
    ))}
  </div>
```

Reply:
Done, thank you very much!

Reply:
Anytime